traveler-ios-ui/TravelerKitUI/ViewControllers/ParkingQueryFormViewController.swift

```diff
@@ -39,8 +39,8 @@ public class ParkingQueryFormViewController: UIViewController {
     public override func viewDidAppear(_ animated: Bool) {
         parkingFilterContext?.addObserver(self)
     }
-
-    public override func viewWillDisappear(_ animated: Bool) {
+    
+    deinit {
```

Comment:
When adding the observer on `viewDidAppear`, remove it on `viewWillDisappear`.  When adding the observer on `viewDidLoad`, remove it in `deinit`